### PR TITLE
Finish week four sentiment analysis

### PR DESCRIPTION
**Pull request by panerorenn9541**
Implement the sentient analysis feature from week four

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -11,66 +11,87 @@
...
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("comment");
-
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
-
+
```
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: remove empty space

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -11,66 +11,87 @@
...
+    Entity commentEntity = new Entity("comment");
+    commentEntity.setProperty("message", text);
+    commentEntity.setProperty("name", name);
+    commentEntity.setProperty("sentiment", score);
+ 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(messageEntity);
-
+    datastore.put(commentEntity);
+
```
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: remove empty space

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -25,17 +25,32 @@ <h2 style="text-align: center;">
...
+
+        <p>How many comments do you want to see?</p>
+        <select id="commentLimit" onchange="loadComments()">
+          <option value="0">0 </option
+          ><option value="1">1 </option
+          ><option value="2">2 </option
+          ><option value="3">3 </option
+          ><option value="4">4 </option
+          ><option value="5">5 </option></select
+        >
```
Comment (huzcn) [inline portfolio/src/main/webapp/index.html]: should this be in the end of last line?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -11,66 +11,87 @@
...
+ 
+    public Comment(long id, String name, String message, String sentiment) {
+      this.id = id;
+      this.name = name;
       this.message = message;
+      this.sentiment = sentiment;
     }
   }
-
+
```
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: remove space?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 5]
```diff
@@ -11,66 +11,87 @@
...
+      long id = entity.getKey().getId();
       String message = (String) entity.getProperty("message");
-      Comment comment = new Comment(message);
+      String name = (String) entity.getProperty("name");
+      String sentiment = (String) entity.getProperty("sentiment");
+      Comment comment = new Comment(id, name, message, sentiment);
       comments.add(comment);
     }
-
+
```
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: remove space

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 6]
```diff
@@ -11,66 +11,87 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps.servlets;
-
+
```
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: remove space

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 7]
```diff
@@ -11,66 +11,87 @@
...
 import com.google.gson.Gson;
-import com.google.sps.data.Comment;
 import java.io.IOException;
 import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
-
+
```
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: empty space

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (huzcn) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: empty space

**Review by huzcn — APPROVED** [review]
Nice. Approved for merging after nits are resolved.

**Comment by panerorenn9541** [comment]
Fix the suggested changes

**Pull request merged**